app/uk/gov/hmrc/lisaapi/config/AppContext.scala

```diff
@@ -21,22 +21,22 @@ import play.api.Configuration
 import uk.gov.hmrc.play.bootstrap.config.{RunMode, ServicesConfig}
 
 @Singleton
-class AppContext @Inject()(runModeConfiguration: Configuration, runMode: RunMode) extends ServicesConfig(runModeConfiguration: Configuration, runMode: RunMode) {
-  lazy val appName = runModeConfiguration.getString("appName").getOrElse(throw new RuntimeException("appName is not configured"))
-  lazy val appUrl = runModeConfiguration.getString("appUrl").getOrElse(throw new RuntimeException("appUrl is not configured"))
+class AppContext @Inject()(config: Configuration, runMode: RunMode) extends ServicesConfig(config, runMode) {
```

Comment:
Services config can be injected instead of being extended so you don't have to bring in both configuration and run mode.

Reply:
makes sense